sdk/python/feast/cli.py

```diff
@@ -118,57 +87,66 @@ def entity_describe(name: str, project: str):
     )
 
 
-@entity.command(name="list")
-@click.option(
-    "--project",
-    "-p",
-    help="Project that entity belongs to",
-    type=click.STRING,
-    default="",
-)
-@click.option(
-    "--labels",
-    "-l",
-    help="Labels to filter for entities",
-    type=click.STRING,
-    default="",
-)
-def entity_list(project: str, labels: str):
+@entities_cmd.command(name="list")
+def entity_list():
     """
     List all entities
     """
-    feast_client = Client()  # type: Client
-
-    labels_dict = _get_labels_dict(labels)
-
+    cli_check_repo(Path.cwd())
+    store = FeatureStore(repo_path=str(Path.cwd()))
     table = []
-    for entity in feast_client.list_entities(project=project, labels=labels_dict):
+    for entity in store.list_entities():
         table.append([entity.name, entity.description, entity.value_type])
 
     from tabulate import tabulate
 
     print(tabulate(table, headers=["NAME", "DESCRIPTION", "TYPE"], tablefmt="plain"))
 
 
-def _get_labels_dict(label_str: str) -> Dict[str, str]:
+@cli.group(name="feature-views")
+def feature_views_cmd():
+    """
+    Access feature views
     """
-    Converts CLI input labels string to dictionary format if provided string is valid.
+    pass
+
+
+@feature_views_cmd.command("describe")
+@click.argument("name", type=click.STRING)
+def feature_view_describe(name: str):
+    """
+    Describe a feature view
+    """
+    cli_check_repo(Path.cwd())
+    store = FeatureStore(repo_path=str(Path.cwd()))
+
+    try:
```

Comment:
Is there a reason to have this try/except?

Reply:
It's a difference between this
```
$ feast entities describe driver_id1
Traceback (most recent call last):
  File "/Users/tsotne/feast/venv/bin/feast", line 33, in <module>
    sys.exit(load_entry_point('feast', 'console_scripts', 'feast')())
  File "/Users/tsotne/feast/venv/lib/python3.7/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/Users/tsotne/feast/venv/lib/python3.7/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/Users/tsotne/feast/venv/lib/python3.7/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/tsotne/feast/venv/lib/python3.7/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/tsotne/feast/venv/lib/python3.7/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/tsotne/feast/venv/lib/python3.7/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/Users/tsotne/feast/sdk/python/feast/cli.py", line 77, in entity_describe
    entity = store.get_entity(name)
  File "/Users/tsotne/feast/sdk/python/feast/feature_store.py", line 146, in get_entity
    return self._registry.get_entity(name, self.project)
  File "/Users/tsotne/feast/sdk/python/feast/registry.py", line 129, in get_entity
    raise EntityNotFoundException(project, name)
feast.errors.EntityNotFoundException: Entity driver_id1 does not exist in project my_repo
```
and this
```
$ feast entities describe driver_id1
Entity driver_id1 does not exist in project my_repo
```

I think having a stack trace is a worse user experience

Reply:
Yip, looks good to me.